Library/Formula/neofetch.rb

```diff
@@ -0,0 +1,31 @@
+class Neofetch < Formula
+    desc "A CLI system information tool written in BASH"
+    homepage "https://github.com/dylanaraps/neofetch"
+    url "https://github.com/dylanaraps/neofetch/archive/refs/heads/master.zip"
```

Comment:
Instead of `master`, could you use the tarball for the version 7.1.0 you've marked this as? https://github.com/dylanaraps/neofetch/archive/refs/tags/7.1.0.tar.gz

Reply:
That seems to break my patches, I'll look into it further and fix.

    ==> Patching
    ==> Applying 993cfbc62776cb6e17def9ec53751b0876312a1d.patch
    patching file neofetch
    Hunk #1 succeeded at 1230 (offset -30 lines).
    Hunk #2 succeeded at 2136 (offset -153 lines).
    Hunk #3 FAILED at 2563.
    1 out of 3 hunks FAILED -- saving rejects to file neofetch.rej
    Error: Failure while executing: /usr/bin/patch -g 0 -f -p1 -i /private/tmp/neofetch--patch20220707-6903-1ajrbu9/993cfbc62776cb6e17def9ec53751b0876312a1d.patch